phpstan.neon.dist

```diff
@@ -4,6 +4,8 @@ includes:
 parameters:
     level: max
     checkMissingIterableValueType: false
+    checkGenericClassInNonGenericObjectType: false
+    treatPhpDocTypesAsCertain: false
```

Comment:
I am not sure.. if this is something we want...

Reply:
The below errors then would get triggered. But, managed to get those fixed in afef0f3e14e80379b6dea9f0b0560252c4c0bd68 to allow PHPStan to do these type of checks as well.

![image](https://user-images.githubusercontent.com/2707563/83874409-b16c6e00-a735-11ea-80b4-ccbdc524ac1c.png)

Reply:
I was talking about `treatPhpDocTypesAsCertain`

Reply:
all the reports above are for `checkGenericClassInNonGenericObjectType` which I am fine setting it to `false`

Reply:
Managed to get all errors resolved, so there is no need to change the PHPStan config anymore 👍